tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -772,8 +773,16 @@ public function theUserCreatesNewUser(string $user, TableNode $table): void {
 	 */
 	public function addUserToGroup(string $group, string $user, ?string $byUser = null): ResponseInterface {
 		$credentials = $this->getAdminOrUserCredentials($byUser);
-		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
-		$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
+		if ($group === "nonexistent") {
+			$groupId = WebDavHelper::generateUUIDv4();
+		} else {
+			$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id");
+		}
+		if ($user === "nonexistent") {
+			$userId = WebDavHelper::generateUUIDv4();
+		} else {
+			$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
+		}
```

Comment:
```suggestion
		$groupId = ($group === "nonexistent") ? WebDavHelper::generateUUIDv4() : $this->featureContext->getAttributeOfCreatedGroup($group, "id");
		$$userId = ($user === "nonexistent") ? WebDavHelper::generateUUIDv4() : $userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
```
can we make short like this?

Reply:
I would prefer if-else block in this case because it's easier to read. But I am fine with both